in main
foo.test() returned: 42
